<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Jokenpo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="btn_stone">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/pedra.png</normaloff>../img/pedra.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>57</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_paper">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/paper1.png</normaloff>../img/paper1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>62</width>
     <height>62</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_scissors">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/tesoura1.png</normaloff>../img/tesoura1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>63</width>
     <height>63</height>
    </size>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_msg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bosch Sans Bold</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="title_jogo_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bosch Sans Black</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Joken</string>
   </property>
  </widget>
  <widget class="QLabel" name="icon_velha_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/flag-japan_1f1ef-1f1f5.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bosch Sans Bold</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(199, 0, 86);</string>
   </property>
   <property name="text">
    <string>Play Again</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-260</x>
     <y>-30</y>
     <width>811</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/japanese-temple-surrounded-by-nature_52683-46009.webp</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>371</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 0.3);</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_jogo_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bosch Sans Black</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(199, 0, 86);</string>
   </property>
   <property name="text">
    <string>Po</string>
   </property>
  </widget>
  <widget class="QLabel" name="jogada">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>graphicsView_2</zorder>
  <zorder>btn_stone</zorder>
  <zorder>btn_paper</zorder>
  <zorder>btn_scissors</zorder>
  <zorder>lbl_msg</zorder>
  <zorder>title_jogo_2</zorder>
  <zorder>icon_velha_2</zorder>
  <zorder>btn_reset</zorder>
  <zorder>title_jogo_3</zorder>
  <zorder>jogada</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
